js/main.js

```diff
@@ -65,3 +66,25 @@ let createTodoElement = (elementId, todoText) => {
   let list = document.getElementById("todoList");
   list.appendChild(todo);
 }
+
+
+let showTime = () => {
+  let date = new Date();
+  let h = date.getHours(); // 0 - 23
+  let m = date.getMinutes(); // 0 - 59
+  let s = date.getSeconds(); // 0 - 59
+
+  
+  h = (h < 10) ? "0" + h : h;
+  m = (m < 10) ? "0" + m : m;
+  s = (s < 10) ? "0" + s : s;
+  
+  let time = h + ":" + m + ":" + s;
+  document.getElementById("clock").innerText = time;
+  document.getElementById("clock").textContent = time;
```

Comment:
Why are you using both innerText and textContent?

Reply:
Very good question, I used this crazy tool "CTRL+C" and "CTRL+V", thought I removed all the unnecessary code, good catch! :p